src/app/shared/services/authentication.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+
+import { User } from '../interfaces/user.interface.';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+
+  constructor() { }
```

Comment:
You may delete constructor if you don`t use it

Reply:
ok, sure